snippets/distr-tracing-tempo-required-secret-parameters.adoc

```diff
@@ -1,7 +1,6 @@
 // Text snippet included in the following modules:
 //
-// * distr-tracing-tempo-install-web-console.adoc
-// * distr-tracing-tempo-install-cli.adoc
+// * distr-tracing-tempo-storage-ref.adoc
```

Comment:
Not a major change but was this change intended?

Reply:
Thank you. Yes, it's just about fixing some outdated metadata.